huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go

```diff
@@ -536,11 +540,85 @@ resource "huaweicloud_networking_secgroup" "test2" {
   delete_default_rules = true
 }
 
+resource "huaweicloud_networking_secgroup" "test3" {
```

Comment:
you can use `count` to create multi-resources

```hcl
resource "huaweicloud_networking_secgroup" "test" {
  count                 = 4
  name                 = "%[2]s-secgroup-${count.index}"
  delete_default_rules = true
}
```

Reply:
sure